js/app/fish.js

```diff
@@ -24,13 +24,17 @@ class Fish {
         fishingSpot = DATA.FISHING_SPOTS[fishData.location];
         spearfishing = false;
       }
-      this.location = {
-        id: fishingSpot._id,
-        name: __p(fishingSpot, "name"),
-        zoneId: fishingSpot.territory_id, /* This is not a typo! */
-        zoneName: __p(DATA.ZONES[DATA.WEATHER_RATES[fishingSpot.territory_id].zone_id], "name"),
-        spearfishing: spearfishing,
-      };
+      if (fishingSpot.territory_id !== 0){
```

Comment:
Was this causing a bug somewhere?  I'm pretty sure none of the territory IDs would be 0

Reply:
Yes it coused one for fishes in the diadem + some spearfisches on 1 specific node.